event_listener/dfusion_db/event_receivers.py

```diff
@@ -75,12 +75,12 @@ class StateTransitionReceiver(GenericEventReceiver):
     def real_save(self, parsed_event: Dict[str, Any], block_info=None):
 
         # Verify integrity of post data
-        assert parsed_event.keys() == {'transitionType', 'stateIndex', 'stateHash', 'slot'}, \
+        assert parsed_event.keys() == {'transitionType', 'from', 'to', 'slot'}, \
             "Unexpected Event Keys: got {}".format(parsed_event.keys())
-        _hash = parsed_event['stateHash']
+        _hash = parsed_event['to']
         _type = parsed_event['transitionType']
 
-        assert isinstance(parsed_event['stateIndex'], int), "Transition to has unexpected values"
+        assert isinstance(parsed_event['from'], int), "Transition to has unexpected values"
```

Comment:
this isn't right... if we are going back to use from then this should be a length 64 string and we need to check that the same is true for 'to'. However I really don't think this is what we want to do.